stuff/cron/plagiarism_detector.py

```diff
@@ -170,23 +170,29 @@ def filter_and_format_result(dbconn: lib.db.Connection, contest_id: int,
 
     updated_result: List[Results] = []
     for result in results:
-        temp_submission_id_1 = guid_and_submission_id_dict[os.path.splitext(
+        submission_id_1 = guid_and_submission_id_dict[os.path.splitext(
             os.path.basename(result.test_filename))[0]]
-        temp_submission_id_2 = guid_and_submission_id_dict[os.path.splitext(
+        submission_id_2 = guid_and_submission_id_dict[os.path.splitext(
             os.path.basename(result.reference_filename))[0]]
-        if (temp_submission_id_1 > temp_submission_id_2):
-            temp_submission_id_1, temp_submission_id_2 = temp_submission_id_2, temp_submission_id_1
+        score_1 = result.test_score
+        score_2 = result.reference_score
+        code_1 = result.highlighted_test_code
+        code_2 = result.highlighted_reference_code
+        if submission_id_1 > submission_id_2):
```

Comment:
```suggestion
        if submission_id_1 > submission_id_2:
```

note that Python conditionals don't use parentheses.